airbyte-api/build.gradle

```diff
@@ -201,4 +197,6 @@ sourceSets {
     }
 }
 
-Task publishArtifactsTask = getPublishArtifactsTask("$rootProject.ext.version", project)
+tasks.named('compileJava').configure {
+    dependsOn tasks.named('generate')
+}
```

Comment:
I now define a per-project `generate` task in the root `gradle.build`. Its purpose is to group all codegen-related tasks.